bowling.test.js

```diff
@@ -0,0 +1,56 @@
+const Bowling = require('./bowling.js');
+
+let bowling;
+
+function multipleRolls(rolls, pins) { // function to roll several times
```

Comment:
Tiny detail on naming here - the name `rolls` make me think that it's an array, rather than an integer. You can check this article I wrote about naming if you'd like: https://medium.com/swlh/an-opinionated-guide-to-naming-your-code-aimed-at-new-developers-6f769d60e098?sk=8176415e3a62db07be59877d61bac7de